pkg/jobs/jobs.go

```diff
@@ -239,12 +239,17 @@ func validationJob(
 		case errors.Is(err, context.Canceled):
 			return nil
 		default:
+			query, prettyErr := stmt.PrettyCQL()
 			globalStatus.AddReadError(&joberror.JobError{
 				Timestamp: time.Now(),
 				StmtType:  stmt.QueryType.ToString(),
 				Message:   "Validation failed: " + err.Error(),
-				Query:     stmt.PrettyCQL(),
+				Query:     query,
 			})
+
+			if prettyErr != nil {
+				return prettyErr
```

Comment:
Let's decide on what to do with prettyCQL error and following this logic everywhere.
We can either return it, to make gemini to fail, which I prefere, or we can log it. as yu did below, in ddl

Reply:
I'll make it fail when it encounters the prettyCQL error